cypress/integration/Checkbox.feature

```diff
@@ -0,0 +1,126 @@
+Feature: Checkbox component
+  I want to change Checkbox, reverse, fieldHelpInline, label, help label, input width, field help, labelInline, labelWidth, labelAlign properties
+
+  Background: Open Checkbox component page
+    Given I open "Checkbox" component page
+
+  @positive
+  Scenario: Enable reverse checkbox
+    When I enable reverse checkbox
+    Then checkbox helpText property is set to "reverse"
+
+  @positive
+  Scenario: Enable and disable reverse checkbox
+    When I enable reverse checkbox
+      And I disable reverse checkbox
+    Then checkbox helpText property is not set to "reverse"
+
+  @positive
+  Scenario: Enable fieldHelpInline
+    When I enable fieldHelpInline
+    Then checkbox helpText property is set to "fieldHelpInline"
+
+  @positive
+  Scenario: Enable and disable fieldHelpInline
+    When I enable fieldHelpInline
+      And I disable fieldHelpInline
+    Then checkbox helpText property is not set to "fieldHelpInline"
+
+  @positive
+  Scenario Outline: Change Checkbox component label
+    When I set label to "<label>"
+    Then Label on preview is "<label>"
```

Comment:
the Label word should starts from small character (it's not a component)

Reply:
fixed